src/components/10-atoms/checkbox/index.scss

```diff
@@ -168,41 +168,41 @@ axa-checkbox {
     }
   }
 
-    &[styled] .a-checkbox__content *,
-    .a-checkbox__children-inline,
-    .a-checkbox__children-inline * {
+  &[styled] .a-checkbox__content *,
+  .a-checkbox__children-inline,
+  .a-checkbox__children-inline * {
       display: inline;
       @include typo(primary, small-1);
 
       @include breakpoint($mediaquery-md-up) {
          @include typo(primary, medium);
       }
-    }
+  }
 
-    &:hover {
+  &:hover.no-touch {
```

Comment:
On touch devices there is no hover effect anymore. nice. 

But the bug still exists (on https://axa-ch.github.io/plib-feature/bugfix/checkbox-uncontrolled/?path=/story/components-atoms-checkbox--checkbox ):
**EDIT: The problem could be, that this feature branch deploy is too old. I saw that azure was failing.**

If you tap on a checked checkbox, there is thas small square inside the checkbox:
![image](https://user-images.githubusercontent.com/7448004/78104097-d1f31180-73ee-11ea-9b9e-7af0a4118bfa.png)

Reply:
Dont know why, but this rule is active:
![image](https://user-images.githubusercontent.com/7448004/78104345-60679300-73ef-11ea-9041-b67a689e6c35.png)

Reply:
I think this is all due to feature-branch problems, and/or not using a real mobile device. Using a local server and real Android phone with Chrome, everything seems to work.